plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -641,23 +641,25 @@ struct RuntimeHistoryTable {
 }
 
 @TableName(mutationScoreHistoryTable)
+@TableConstraint("time UNIQUE (time)")
 struct MutationScoreHistoryTable {
     long id;
 
     /// when the measurement was taken.
     @ColumnName("time")
-    SysTime timeStamp;
+    string timeStamp;
```

Comment:
It is preferable to keep it as a SysTime because, it is time. type safety :)
Have you considered that instead of changing it to a string changing how the systime is saved? such that it is still a SysTime but hours:minues:seconds:usecs are set to zero, always, when saving to the database. That togethered with a unique constraint may be good enough?